cmd/kubeadm/app/preflight/checks.go

```diff
@@ -182,6 +184,38 @@ func (fac FileAvailableCheck) Check() (warnings, errors []error) {
 	return nil, errors
 }
 
+// FileContentCheck checks that the given file contains the string Content.
+type FileContentCheck struct {
+	Path    string
+	Content []byte
+}
+
+func (fcc FileContentCheck) Check() (warnings, errors []error) {
+	errors = []error{}
```

Comment:
I don't think this variable has any function here as you're directly returning the error array when it has got one object (it will never contain 2 errors)